BaseFilter/sections/specific.txt

```diff
@@ -275,6 +275,7 @@ chortle.co.uk,coloring.ws,dltk-holidays.com,dltk-kids.com,kidzone.ws,pcsteps.com
 !
 !
 !
+merriam-webster.com##div[id^='mw-ad-']
```

Comment:
Please always check with AdGuard app. Your rule and Easylists are not applied because of `display: flex!important;` style.
You can make selector of original rule(you can find that rule in AG Extension's filtering log) more specific, and the rule will work:
`merriam-webster.com##.abl`
to
`merriam-webster.com##body .abl`